setup.py

```diff
@@ -29,6 +29,9 @@ def read(fname):
         'templates/trellis/trellis-makefile.j2',
         'templates/ascentlint/Makefile.j2',
         'templates/ascentlint/run-ascentlint.tcl.j2',
+        'templates/symbiflow/symbiflow-makefile.j2',
+        'templates/yosys/yosys-makefile.j2',
+        'templates/yosys/yosys-script-tcl.j2',
```

Comment:
Did you catch all of the new templates? Weren't there additional Yosys templates for Vivado?

Reply:
Templates added